### PR TITLE
Create Assignment_1.cu

### PR DESCRIPTION
**Pull request by tharun634**

### CODE REVIEW

File: Assignment_1.cu [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    int id = blockIdx.x * blockDim.x + threadIdx.x;
+    if(id < *d_array_size)
+     d_out[i] = d1_in[i] + d2_in[i];    
+}
+int main()
+{
+    cout << "Enter the array size : ";
+    cin >> Array_Size;
+	
+    float h_in1[Array_Size],h_in2[Array_Size] h_out[Array_Size];
```
Comment (ameyanrd) [inline Assignment_1.cu]: float h_in1[Array_Size],h_in2[Array_Size], h_out[Array_Size]; // You forgot the comma after h_in2.

**Review by ameyanrd — CHANGES_REQUESTED** [review]
Hi! Please make the changes, compile your program and push the changes.

File: Assignment_1.cu [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+using namespace std;
+int Array_Size;
+
+__global__ void Sum(float* d_in1,float* d_in2, float* d_out,int* d_array_size)
+{
+    int id = blockIdx.x * blockDim.x + threadIdx.x;
+    if(id < *d_array_size)
+     d_out[i] = d1_in[i] + d2_in[i];
```
Comment (ameyanrd) [inline Assignment_1.cu]: error: identifier "i" is undefined
error: identifier "d1_in" is undefined
error: identifier "d2_in" is undefined

**Review by ameyanrd — APPROVED** [review]
Great!

**Review by niranjansy — APPROVED** [review]

**Pull request merged**